Code_Exercises/Exercise_02_First_Kernel/solution.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * SYCL Quick Reference
+ * ~~~~~~~~~~~~~~~~~~~~
+ *
+ * // Include SYCL header
+ * #include <CL/sycl.hpp>
+ *
+ * // Default construct a queue
+ * auto q = sycl::queue{};
+ *
+ * // Allocate device memory
+ * auto * devPtr = malloc_device<T>(mycount, q);
```

Comment:
It isn't obvious where T comes from for this. Is it better to just use <sizeof(int)> if we are defining ints in the file for the person to use in this exercise?

Reply:
Sure change made